templates/extra/privacy.html

```diff
@@ -16,8 +16,8 @@ <h1 class="privacy-title">Privacy</h1>
         <p>While using our Service, we may ask you to provide us with certain personally identifiable information that can be used to contact or identify you. Personally identifiable information may include, but is not limited to, your email address, name, phone number, postal address, other information ("Personal Information").</p>
         <p>We collect this information for the purpose of providing the Service, identifying and communicating with you, responding to your requests/inquiries, servicing your purchase orders, and improving our services.</p>
         <h5>Log Data</h5>
-        <p>We collect information that your browser sends whenever you visit our Service ("Log Data"). This Log Data may include information such as your computer's Internet Protocol ("IP") address, browser type, browser version, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages and other statistics.</p>
-        <p>In addition, we may use third party services such as Google Analytics that collect, monitor and analyze this type of information in order to increase our Service's functionality. These third party service providers have their own privacy policies addressing how they use such information.</p>
+        <p>We collect information that your browser sends whenever you visit our Service ("Log Data"). This Log Data may include information such as your computer's Internet Protocol ("IP") address, Internet Service Provider, referring URL, browser type, browser version, operating system, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages, links clicked, non-sensitive text entered, mouse movements, cookie information, and other statistics.</p>
+        <p>In addition, we may use third party services such as Google Analytics that collect, monitor and analyze this type of information in order to increase our Service's functionality. We may collect information from such visitor regarding his use of that Website, such as pages visited, links clicked, non-sensitive text entered, and mouse movements, as well as information more commonly collected such as his IP address, referring URL, browser, operating system, cookie information, and Internet Service Provider ("Visitors' Information”). These third party service providers have their own privacy policies addressing how they use such information.</p>
```

Comment:
This line should be reverted since we added the clauses needed in the line before.